src/MongoDB.Driver.Core/Core/Operations/CountDocumentsOperation.cs

```diff
@@ -27,6 +27,7 @@ namespace MongoDB.Driver.Core.Operations
 {
     /// <summary>
     /// Represents a count documents operation.
+    /// For a fast count of the total documents in a collection see <see class="EstimatedDocumentCountOperation"/>.
```

Comment:
Do we really expect that it will be helpful for a user?

Reply:
I think we should not promise a "fast count". How about:

    // For a fast estimate of the total number of documents in a collection ...

Reply:
To Dmitry's point, this is in Core and most users don't use Core directly.

Reply:
Re: wording:
I'm happy with the word "estimate" over count. Playing devil's advocate, I suppose the name "estimatedDocumentCount" does strongly imply "estimate"
The [drivers ticket](https://jira.mongodb.org/browse/DRIVERS-630) does unfortunately say "MUST" add the following wording.
Given the above, do you still feel like we should still change the wording? I don't mind either way.

Reply:
Re: Core.
Agreed, this class is in Core, and most users don't use Core directly. The case for documenting this class would just be that it's still technically part of the public API (just a portion that most users don't use).  I'm leaning towards keeping the comment as I don't think there's any harm in adding it

Reply:
No problem adding comments in Core.

Reply:
So the DRIVERS ticket is forcing us to use inaccurate language?

How did the DRIVERS ticket get approved that way?

I'll defer to @jyemin whether we should use accurate language or the language from the DRIVERS ticket.

Reply:
The DRIVERS ticket says "Suggested text:" so it's ok to improve on it.  (There is no formal approval process for DRIVERS tickets that don't have an actual spec change associated with it).  Please add a comment to that ticket with your objections to the suggested language, as many drivers haven't made this change yet.

Reply:
I added a comment to the DRIVERS ticket.

Reply:
Oops, Jeff is correct, I misread the ticket: the wording is indeed suggested, the "MUST" is for adding text. Either way, Bernie replied to your suggestion saying it was fine, so I took Robert's updated wording! :)